packages/langium/tsconfig-eslint.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Why did you add this file? What is is used for?

Reply:
It is used in .eslintrc.json in `project` section to be when checking the `test` folder. Otherwise running `eslint src test` produces errors like `Can't read tsconfig.json` wenn analyzing the test files.

Reply:
One could probably reduce the content to this one:
```json
{
  "extends": "./tsconfig.json",
  "include": [
    "test/**/*.ts"
  ]
}

```

Reply:
Oh I see, it's because of this:

> Error: Error while loading rule '@typescript-eslint/no-floating-promises': You have used a rule which requires parserServices to be generated. You must therefore provide a value for the "parserOptions.project" property for @typescript-eslint/parser.

Reply:
Yes, this rule needs Types information, otherwise it is almost useless